src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-wizard/rgw-multisite-wizard.component.scss

```diff
@@ -0,0 +1,11 @@
+.container-fluid {
+  align-items: flex-start;
+  display: flex;
+  padding-left: 0;
+  width: 100%;
+}
+
+::ng-deep .custom-modal-content .modal-content {
+  right: 40vh;
+  width: 140vh;
+}
```

Comment:
why is this customization? and this way of overriding styles eventually will cause issues in downstream. if the `lg` is not enough why not go for an `xl` modal?
https://getbootstrap.com/docs/5.0/components/modal/#optional-sizes

Reply:
`xl` is also not enough for this modal, so had to go with this custom width